mod_reforged/hooks/skills/actives/charm_skill.nut

```diff
@@ -0,0 +1,21 @@
+::Reforged.HooksMod.hook("scripts/skills/actives/charm_skill", function(q) {
+	q.create = @(__original) function()
+	{
+		__original();
+		// Vanilla is missing a description for this skill
+		this.m.Description = ::Reforged.Mod.Tooltips.parseString("Try to charm a character, forcing him to turn on his allies and obey you instead. The higher a character\'s [Resolve|Concept.Bravery], the higher the chance to resist being charmed.");
+	}
+
+	// Vanilla doesn't have a getTooltip function defined for this skill
+	q.getTooltip = @() function()
+	{
+		local ret = this.skill.getTooltip();
```

Comment:
This will not display AP and Fatigue cost. Use `getDefaultUtilityTooltip` instead.